Ontology/Asset/Equipment/HVACEquipment/AirHandlingUnit.json

```diff
@@ -300,25 +216,7 @@
         "en": "secondary cooling method"
       },
       "name": "secondaryCoolingMethod",
-      "dtmi:dtdl:property:schema;2": {
-        "@type": "Enum",
-        "dtmi:dtdl:property:enumValues;2": [
-          {
-            "enumValue": "ChilledWater",
-            "name": "ChilledWater"
-          },
-          {
-            "enumValue": "DX",
-            "name": "DX"
-          },
-          {
-            "enumValue": "None",
-            "name": "None"
-          }
-        ],
-        "valueSchema": "string"
-      },
-      "writable": true
+      "schema": "dtmi:digitaltwins:rec_3_3:asset:HVACCoolingMethod;1"
```

Comment:
A Property cannot have a schema that is an interface. This element could be changed to be a Component or the schema defined inline or in a reusable schema in the Interface's schemas element.